src/main/java/at/hannibal2/skyhanni/config/features/InventoryConfig.java

```diff
@@ -415,5 +415,10 @@ public static class TiaRelayConfig {
     @FeatureToggle
     public boolean highlightAuctions = true;
 
+    @Expose
+    @ConfigOption(name = "Shift Click For Equipment", desc = "Makes normal clicks to shift clicks in equipment inventory")
```

Comment:
Please remove the word "for" in the option name, the text gets too small :/ 
![image](https://github.com/hannibal002/SkyHanni/assets/24389977/e368dc3a-dbff-4f9d-a71a-13c31e34aa32)